program_transformers/src/token_metadata/master_edition.rs

```diff
@@ -85,14 +75,47 @@ pub async fn save_master_edition(
         ..Default::default()
     };
 
-    if let Some((_me, Some(asset))) = master_edition {
-        let mut updatable: asset::ActiveModel = asset.into();
-        updatable.supply = ActiveValue::Set(Decimal::from(1));
-        updatable.specification_asset_class = ActiveValue::Set(Some(SpecificationAssetClass::Nft));
```

Comment:
This is strange I don't understand why this used to be here.  Seems really extra.

Reply:
yeah, this should be done when metadata acc is created or updated